.github/workflows/beam_PostCommit_Java_IO_GCS_Performance_Tests.yml

```diff
@@ -0,0 +1,122 @@
+  - cron: '0 */6 * * *'
+  workflow_dispatch:
+
+#Setting explicit permissions for the action to avoid the default permissions which are `write-all` in case of pull_request_target event
+permissions:
+  actions: write
+  pull-requests: read
+  checks: read
+  contents: read
+  deployments: read
+  id-token: none
+  issues: read
+  discussions: read
+  packages: read
+  pages: read
+  repository-projects: read
+  security-events: read
+  statuses: read
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.sha || github.head_ref || github.ref }}-${{ github.event.sender.login }}-${{ github.event.schedule }}'
+  cancel-in-progress: true
+
+env:
+  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
+  GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GE_CACHE_USERNAME }}
+  GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GE_CACHE_PASSWORD }}
+
+jobs:
+  beam_PostCommit_Java_IO_GCS_Performance_Tests:
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'workflow_dispatch' ||
+      github.event_name == 'schedule' ||
+      github.event.comment.body == 'Run Java PostCommit IO GCS Performance Tests'
+#    runs-on: [self-hosted, ubuntu-20.04, main]
+    runs-on: ubuntu-20.04
+    name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    strategy:
+      matrix:
+        job_name: ["beam_PostCommit_Java_IO_GCS_Performance_Tests"]
+        job_phrase: ["Run Java PostCommit IO GCS Performance Tests"]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup repository
+      uses: ./.github/actions/setup-action
+      with:
+        comment_phrase: ${{ matrix.job_phrase }}
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    - name: Checkout release branch
+      if: github.event_name == 'schedule' #This has scheduled runs run against the latest release
+      uses: actions/checkout@v3
+      with:
+        ref: release-2.50.0 #TODO automate updating this
+        repository: apache/beam
+    - name: Checkout non-release branch
+      if: github.event_name != 'schedule' #This has triggered runs checkout the triggering branch
+      uses: actions/checkout@v3
+    - name: Install Java
+      uses: actions/setup-java@v3.8.0
+      with:
+        distribution: 'zulu'
+        java-version: '8'
+    - name: Setup Gradle
+      uses: gradle/gradle-build-action@v2
+      with:
+        cache-read-only: false
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Authenticate on GCP
+      uses: google-github-actions/setup-gcloud@v0
+      with:
+        service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
+        project_id: ${{ secrets.GCP_PROJECT_ID }}
+        export_default_credentials: true
+    - name: GCloud Docker credential helper
```

Comment:
Are you actually publishing to docker? If not, you don't need this or the docker buildx step